x-pack/plugins/apm/public/components/app/mobile/transaction_overview/index.tsx

```diff
@@ -88,15 +89,14 @@ export function MobileTransactionOverview() {
       />
       <EuiSpacer size="s" />
       <EuiPanel hasBorder={true}>
-        <TransactionsTable
-          hideViewTransactionsLink
-          numberOfTransactionsPerPage={25}
-          showMaxTransactionGroupsExceededWarning
+        <TransactionOverviewTabs
           environment={environment}
-          kuery={kueryWithMobileFilters}
+          kueryWithMobileFilters={kueryWithMobileFilters}
```

Comment:
For consistency, we might want to change that to `kuery={kueryWithMobileFilters}` as we use `kuery` property everywhere

Reply:
[4442319](https://github.com/elastic/kibana/pull/153044/commits/4442319614d74bf55387e768a31555127cb7bebe)